build.gradle

```diff
@@ -29,39 +29,13 @@ group = 'org.logstash.dissect'
 version "${new File("VERSION").text.trim()}"
 description = """Dissect Filter, a Logstash plugin - the Java parts"""
 
-project.sourceCompatibility = 1.8
-
-compileJava.options.encoding = 'UTF-8'
-
-task sourcesJar(type: Jar, dependsOn: classes) {
```

Comment:
Note for reviewer:
From Gradle 6 the `javadocJar` and `sourcesJar` is provided by the Java plugin, removing the necessity to create custom tasks and listing jar artifacts to get that job done.